src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -820,6 +820,13 @@ private async Task UpdatePullRequestAsync(InProgressPullRequest pr, List<UpdateA
             _logger.LogInformation("Found {count} required updates for Pull Request {url}", targetRepositoryUpdates.RequiredUpdates.Count, pr.Url);
 
             pr.RequiredUpdates = MergeExistingWithIncomingUpdates(pr.RequiredUpdates, targetRepositoryUpdates.RequiredUpdates);
+
+            if (pr.RequiredUpdates.Count < 1)
+            {
+                _logger.LogInformation("No new updates found for Pull Request {url}", pr.Url);
```

Comment:
What is the difference between this check for updates versus the one on line `814` above? Since they seem to have almost the same log message

Reply:
Line 822

Reply:
The updates are always counted the same but some may have been updated in the PR already